pyi.py

```diff
@@ -163,45 +164,47 @@ def visit_AnnAssign(self, node: ast.AnnAssign) -> None:
             elif node.value and not self._in_class:
                 self.error(node.value, Y092)
 
-    def _check_union_members(self, members: Sequence[ast.AST]) -> None:
-        members_by_unparsed = {}
-        for member in members:
-            members_by_unparsed.setdefault(ast.unparse(member), []).append(member)
+    if sys.version_info >= (3, 9):  # ast.unparse exists
```

Comment:
This part of the diff is a bit unreadable. It indents the code added in #50 and puts it under `if sys.version_info >= (3, 9)`. My `_check_union_member` code uses `ast.unparse` for error reporting, but `ast.unparse` is new in Python 3.9. I didn't realize that when I made #50 and only ran `python3 setup.py test` locally in one virtualenv. Not checking union members isn't great, but it's definitely better than `AttributeError`, and can be improved in future PRs.

Reply:
I wonder if we can use `ast.dump` instead. It's not pretty but should do the job of checking for an equivalent AST.

Reply:
I made it work on Python <3.9 too. The error message is slightly different: on 3.9+ you get `duplicate union member "int"`, otherwise you get `duplicate union member`.